src/python/turicreate/__init__.py

```diff
@@ -16,24 +16,27 @@
 __version__ = '{{VERSION_STRING}}'
 from turicreate.version_info import __version__
 
+from turicreate._deps import LazyModuleLoader
```

Comment:
You should hide this import, i.e. `from turicreate._deps import _LazyModuleLoader`.

Reply:
will do

Reply:
`_deps` itself is already hidden, so it would also be fine to do `from turicreate._deps import LazyModuleLoader as _LazyModuleLoader`.